### PR TITLE
Add EMB parameters details in kube_deploy_nodes.yaml file + renamed heading "Configuring Docker proxy" to "Configuring proxy for Docker"

### PR DESCRIPTION
**Pull request by sunithasomanna**
In the Getting Started Readme, added new EMB parameters in kube yaml file and their descriptions in the odim-controller config paramters table

### CODE REVIEW

**Review by shivacharanms — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2364,6 +2370,8 @@ The following table lists all the configuration parameters required by odim-cont
 |haDeploymentEnabled|When set to true, it deploys third-party services as a three-instance cluster. By default, it is set to true.|
 |connectionMethodConf|Parameters of type array required to configure the supported connection methods. <br><blockquote>NOTE: To deploy a plugin after deploying the resource aggregator services, add its connection method information in the array and update the file using odim-controller `--upgrade` option.<br></blockquote>|
 |kafkaNodePort|The port to be used for accessing the Kafka services from external services. The default port is 30092. You can optionally change it.<br><blockquote>NOTE: Ensure that the port is in the range of 30000 to 32767.<br></blockquote>|
+|MessageBusType|Event message bus type. It is either Kafka or RedisStream.|
```
Comment (shivacharanms) [inline README.md]: Please add a condition in brackets 'These values are case sensitive and should be as given'

**Review by shivacharanms — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sunithasomanna) [reply README.md]: done

**Review by shivacharanms — APPROVED** [review]

**Review by arahimanshaik — APPROVED** [review]

**Review by irappa-pattar — APPROVED** [review]

**Review by jeevan-kamkar — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -2364,6 +2370,8 @@ The following table lists all the configuration parameters required by odim-cont
 |haDeploymentEnabled|When set to true, it deploys third-party services as a three-instance cluster. By default, it is set to true.|
 |connectionMethodConf|Parameters of type array required to configure the supported connection methods. <br><blockquote>NOTE: To deploy a plugin after deploying the resource aggregator services, add its connection method information in the array and update the file using odim-controller `--upgrade` option.<br></blockquote>|
 |kafkaNodePort|The port to be used for accessing the Kafka services from external services. The default port is 30092. You can optionally change it.<br><blockquote>NOTE: Ensure that the port is in the range of 30000 to 32767.<br></blockquote>|
+|MessageBusType|Event message bus type. The value is either `Kafka` or `RedisStreams` and they are case-sensitive.|
+|MessageBusQueue|Event message bus queue name.|
```
Comment (shivacharanms) [inline README.md]: The value should not have spaces.

Comment (irappa-pattar) [reply README.md]: Allowed characters for topic creations are alphabets, numbers, period, underscore, and hyphen.

**Review by shivacharanms — APPROVED** [review]

**Pull request merged**